Framework/CurveFitting/src/CostFunctions/CostFuncFitting.cpp

```diff
@@ -128,7 +128,6 @@ void CostFuncFitting::calActiveCovarianceMatrix(GSLMatrix &covar,
   // construct the jacobian
   GSLJacobian J(*m_function, m_values->size());
   size_t na = this->nParams(); // number of active parameters
-  auto j = J.getJ();
   assert(j->size2 == na);
```

Comment:
I think this might need removing as well now `j` is gone..?

Reply:
I reverted this in a future commit, but still didn't use it correctly. I've fixed this properly now.